libraries/AP_Scripting/scripts/sandbox.lua

```diff
@@ -30,7 +30,7 @@ function get_sandbox_env ()
                    codepoint = utf8.codepoint, len = utf8.len, offsets = utf8.offsets},
 
           -- ArduPilot specific
-          gcs = { send_text = gcs.send_text},
+          millis = millis,
```

Comment:
Is this a misprint?
`millis = millis`

Reply:
No, it's a table constructor, it's assigning into the millis member of the table the existing global value for millis.

In a simpler example (taken from PIL https://www.lua.org/pil/3.6.html )
`a = {x=0, y=0}` is equivalent to `a = {}; a.x=0; a.y=0`